### DISCUSSION TITLE
Exploratory CNV results and seeking general feedback on Wilms tumor annotation (SCPCP000014)

### DISCUSSION THREAD

**Discussion by JingxuanChen7** [AlexsLemonade/OpenScPCA-analysis#808] [Results Q&A]
Related proposed analysis: https://github.com/AlexsLemonade/OpenScPCA-analysis/discussions/628

Dear community, I'm opening this discussion in order to get some feedback on my preliminary results of CNV identification & general suggestions on next steps.

In my last PR (https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/784), I applied Seurat anchor transfer strategy to investigate the general cell type components in all 10 samples. Core result [plots](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/cell-type-wilms-tumor-14/plots/01_anchor_transfer_seurat) are accessible in the repo. In most samples, majority of cells are classified as `fetal_nephron`. In some samples, `fetal_nephron` is even the only one compartment (such as `SCPCL000847`, `SCPCL000851`). 

In my proposal, I planned to identify tumor cells from normal cells, which could also help improve the annotation for normal cells. To start with, I tried [`inferCNV`](https://github.com/broadinstitute/infercnv) and [`copyKat`](https://github.com/navinlabcode/copykat) using sample `SCPCL000850`, which shows more divergent cell compartments, and a relatively clearer annotation compared with others.
![image](https://github.com/user-attachments/assets/c60e6bf4-d462-4454-b51e-bbcb33e289d1)
![image](https://github.com/user-attachments/assets/155833b3-7e14-47c6-84ea-4daba229956a)

## Exploratory CNV results
Both strategies ask for a reference level/set of known normal cells. Here, I selected stroma compartment as the "normal" reference.

#### inferCNV outputs
![image](https://github.com/user-attachments/assets/9f853042-fc3f-4d8b-8c58-c42d16e93cb3)

- We could observe some different CNV profiles between "fetal_nephron" and "immune", such as chr1, chr6, chr11 etc.
- I think it's hard to observe any different CNV profiles within the "fetal_nephron" subgroups. Ideally, I would like to see at least two sub groups (i.e, normal nephrons & tumor cells)

To look more into the inferCNV results, I tried three ways to summarize the output.
- Based on [this](https://www.biostars.org/p/9573777/) Biostars discussion, I calculated a "cnv_score" indicating the proportion of CNVs among all genes.
```
apply(infercnv_out@expr.data, 2 ,function(x){ sum(x < 0.95 | x > 1.05)/length(x) })
```
- If normal/tumor groups are clear, we should expect a bimodal distribution. However, we couldn't observe the bimodal distribution here. (reference = gray)
![image](https://github.com/user-attachments/assets/b10d7b7e-2361-48fc-a100-afb53268209f)
![image](https://github.com/user-attachments/assets/3385c0bc-e31f-4867-8b11-eb4e0f54bf48)

- Based on the implementation in a previous module - [`cell-type-ewings`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/cell-type-ewings/template_notebooks/cnv-workflow), I also tried to calculate the number of CNV chrs & weighted means of scaled CNV proportion.
- I think no clear patterns can be observed, either. (reference = gray)
![image](https://github.com/user-attachments/assets/1b5c2ddb-ca96-445d-8c0f-36cf1e2b4b92)
![image](https://github.com/user-attachments/assets/a2b724d6-c7e9-43ae-81e4-89b8c2a45d37)

- InferCNV also runs extremely slow in my case (~1hr)

#### CopyKat outputs
- Run CopyKat using stroma cells as reference level. This result looks fine (?), since stroma and immune cells are mostly diploid, while the nephron cluster contains plenty of aneuploid (potential tumor) cells.

![image](https://github.com/user-attachments/assets/46f27ea7-5ebe-4196-91db-6f6058cbef85)

- But the heatmap from CopyKat show no big differences in terms of CNV profile.
![image](https://github.com/user-attachments/assets/7d10899d-5c45-4aa2-9f03-42690530e6cd)

<details>
  <summary> Also attach STDOUT message here. There seems a "low data quality" warning message? </summary>

```
[1] "running copykat v1.1.0"
[1] "step1: read and filter data ..."
[1] "40316 genes, 2943 cells in raw data"
[1] "4170 genes past LOW.DR filtering"
[1] "WARNING: low data quality; assigned LOW.DR to UP.DR..."
[1] "step 2: annotations gene coordinates ..."
[1] "start annotation ..."
[1] "step 3: smoothing data with dlm ..."
[1] "step 4: measuring baselines ..."
[1] "93 known normal cells found in dataset"
[1] "run with known normal..."
[1] "baseline is from known input"
[1] "step 5: segmentation..."
[1] "step 6: convert to genomic bins..."
[1] "step 7: adjust baseline ..."
[1] "step 8: final prediction ..."
[1] "step 9: saving results..."
[1] "step 10: ploting heatmap ..."
Time difference of 8.360636 mins
```

</details>

- Then I also tried CopyCat without specifying normal cells, which generated an opposite result to above!!
![image](https://github.com/user-attachments/assets/d87cb745-505e-4016-9e48-885dbc09c454)

![image](https://github.com/user-attachments/assets/8a241937-0ad9-4770-98ca-c484ad9df782)

<details>
  <summary> STDOUT message shows plenty of `WARNING! NOT CONVERGENT!`, indicating a low confidence. </summary>

```
[1] "running copykat v1.1.0"
[1] "step1: read and filter data ..."
[1] "40316 genes, 2943 cells in raw data"
[1] "4170 genes past LOW.DR filtering"
[1] "WARNING: low data quality; assigned LOW.DR to UP.DR..."
[1] "step 2: annotations gene coordinates ..."
[1] "start annotation ..."
[1] "step 3: smoothing data with dlm ..."
[1] "step 4: measuring baselines ..."
number of iterations= 218 
WARNING! NOT CONVERGENT! 
number of iterations= 5000 
number of iterations= 1831 
number of iterations= 98 
number of iterations= 113 
number of iterations= 205 
[1] "low confidence in classification"
[1] "cell: 1"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 2"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 3"
number of iterations= 412 
[1] "cell: 4"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 5"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 6"
number of iterations= 355 
[1] "cell: 7"
number of iterations= 83 
[1] "cell: 8"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 9"
number of iterations= 402 
[1] "cell: 10"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 11"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 12"
number of iterations= 486 
[1] "cell: 13"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 14"
number of iterations= 344 
[1] "cell: 15"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 16"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 17"
number of iterations= 234 
[1] "cell: 18"
number of iterations= 226 
[1] "cell: 19"
number of iterations= 457 
[1] "cell: 20"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 21"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 22"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 23"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 24"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 25"
number of iterations= 312 
[1] "cell: 26"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 27"
number of iterations= 418 
[1] "cell: 28"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 29"
number of iterations= 263 
[1] "cell: 30"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 31"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 32"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 33"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 34"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "cell: 35"
WARNING! NOT CONVERGENT! 
number of iterations= 500 
[1] "step 5: segmentation..."
[1] "step 6: convert to genomic bins..."
[1] "step 7: adjust baseline ..."
[1] "step 8: final prediction ..."
[1] "step 9: saving results..."
[1] "step 10: ploting heatmap ..."
Time difference of 9.10122 mins
```

</details>

#### My questions/concerns
- In conclusion, I think neither `inferCNV` or `CopyKat` could produce solid results to identify tumor cells. I was wondering if there were any issues with my workflow?
- Based on the literature [[1]](https://www.nature.com/articles/s41467-023-43730-0/figures/2) and [[2]](https://www.nature.com/articles/s41416-023-02538-x/figures/1), it looks that CNV is not prevalent in all Wilms tumor patients, so it may not always be a good idea to identify tumor cells based on CNV?
- As I mentioned above, `fetal_nephron` is the only one compartment identified in some samples (such as `SCPCL000847`, `SCPCL000851`). In this case, it's pretty hard to specify a reference level for known normal cells. Identifying CNV in a merged dataset also seems to be pretty tricky. Any suggestions on such samples?

## Other possible next steps
- I am also exploring strategies to improve the clustering using the [clustering module](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/feature/ropenscpca/packages/rOpenScPCA/R/calculate-clusters.R) in this repository. I'd like to try out different [feature selection strategies](https://elifesciences.org/articles/48994) as part of clustering, which worked fine with my colleague.
-  As [cell-type-wilms-tumor-06](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/cell-type-wilms-tumor-06) module applied a similar approach as anchor transfer but different workflows (`SCTransform` + `Azimuth`), it may be worthwhile to add `SCTransform` and/or `Azimuth` as functions to this module. However, since the rationale and reference database are same, I wouldn't expect major differences in the result. Any suggestions?
- Any other suggestions/comments on next steps would be higly appreciated. Many thanks to the team in advance!

**Comment by jashapiro** [comment]
Hi @JingxuanChen7, and thanks for filing this discussion topic. I hope that we are able to get some good feedback!

First, I want to say that it looks like you have done a good amount of work for this initial exploration; we might want to capture some of this in the repository as a notebook, filed in an `exploratory-analysis` directory like we did in [cell-type-ewings/exploratory_analysis](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/cell-type-ewings/exploratory_analysis). Even if this analysis does not end up fully included as a core part of the module, having the exploratory (and possibly negative) results can be quite helpful in addition to this discussion.

I think you may be right to conclude that `infer_cnv` and `CopyKat` are not sufficiently reliable in this context. With CopyKat in particular, I tend to think that the summary calls may not to be the most useful, but diving into the details of the copy number calls across regions can be done, as we did in the Ewings module: [03-copykat.Rmd](https://github.com/maud-p/OpenScPCA-analysis/blob/main/analyses/cell-type-ewings/template_notebooks/cnv-workflow/03-copykat.Rmd). (I am not particularly worried about the "low quality" warning from copykat; I think at that stage it is mostly filtering out genes that do not have significant levels of expression to use in its analysis.)

There was one other method that we started to look at, but might deserve a longer look: [SCEVAN](https://github.com/AntonioDeFalco/SCEVAN) You can see a bit about our experience with it here: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/403#issuecomment-2117700108; Note that one of the issues we encoutered, filtering duplicate gene names, does seem to have been [fixed](https://github.com/AntonioDeFalco/SCEVAN/issues/111).

I am not really sure what to do in the case where you do not have a good normal reference within the sample. It seems like you are using the compartment level annotations here: were there cells that you thought might be reliably normal with a more specific annotation? It may be that the answer is no! If the tumor sample is relatively pure, there may not be much of a baseline to compare against within the sample. I'd like to think there could be a good way to compare across samples to allow the incorporation of additional normal expression profiles, but I have not yet seen one that I can suggest.

But even if copy number estimates were reliable, it might not be the best approach. As you note, aneuploidy may not be a universal or even particularly common attribute in Wilms tumor patients, so while we might regard the clear presence of a somatic CNV as denoting tumor cells, we would not want to assume that euploid cells are not tumor cells.

Are there particular marker genes that you might be able to use to identify the Wilms tumor cells? 


----

For your other directions, I do have a couple of thoughts:

- I think exploring whether the normalization method makes a difference in your analysis could be worth doing; using `SCTransform` could be a worthwhile comparison. I would not necessarily try to use `Azimuth` directly though, as we have found that its extra overhead and complexity may cause more challenges than using the more direct anchor transfer functions. I also would not expect major differences, but it would be worth confirming that!
- For clustering analysis, I think it would be helpful to first define what your goals in the exploration are. What kinds of methods/parameters you want to explore? How do you plan to determine which clusters are reliable?

**Comment by JingxuanChen7** [comment]
Hi @jashapiro , thank you so much the prompt response!

> First, I want to say that it looks like you have done a good amount of work for this initial exploration; we might want to capture some of this in the repository as a notebook, filed in an exploratory-analysis directory like we did in [cell-type-ewings/exploratory_analysis](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/cell-type-ewings/exploratory_analysis). Even if this analysis does not end up fully included as a core part of the module, having the exploratory (and possibly negative) results can be quite helpful in addition to this discussion.

Converting my code/result in this discussion to an Rmd notebook is a great idea, as the results shown here are most negative. I'll work on this in my next PR.

> I think you may be right to conclude that infer_cnv and CopyKat are not sufficiently reliable in this context. With CopyKat in particular, I tend to think that the summary calls may not to be the most useful, but diving into the details of the copy number calls across regions can be done, as we did in the Ewings module: [03-copykat.Rmd](https://github.com/maud-p/OpenScPCA-analysis/blob/main/analyses/cell-type-ewings/template_notebooks/cnv-workflow/03-copykat.Rmd). (I am not particularly worried about the "low quality" warning from copykat; I think at that stage it is mostly filtering out genes that do not have significant levels of expression to use in its analysis.)

Thank you for letting me know that the "low quality warning" should be fine. Regarding copyKat output, I realized that I missed the heatmap, which seemed not as clear as the previous ewings output. No big CNV difference could be found between "aneuploid" and "diploid".

![image](https://github.com/user-attachments/assets/e9815905-8e80-4c2a-affc-2d2b0e5b8502)

> There was one other method that we started to look at, but might deserve a longer look: [SCEVAN](https://github.com/AntonioDeFalco/SCEVAN) You can see a bit about our experience with it here: https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/403#issuecomment-2117700108; Note that one of the issues we encoutered, filtering duplicate gene names, does seem to have been https://github.com/AntonioDeFalco/SCEVAN/issues/111.
 
Really appreciate the suggestion on the new software! I will definitely look into it if time permits.

> Are there particular marker genes that you might be able to use to identify the Wilms tumor cells?

That's a very good point! In the preliminary analysis, I did try to find and marker genes either from literature ([[1]](https://pubmed.ncbi.nlm.nih.gov/35480093/), [[2]](https://pubmed.ncbi.nlm.nih.gov/30885698/), [[3]](https://pubmed.ncbi.nlm.nih.gov/38937666/)) or inferring from DE gene analysis on available wilms tumor dataset ([[4]](https://www.science.org/doi/10.1126/science.aat1699)), and try to explore with dotplots. However, it looks that a lot of the markers are either not expressed, or prevalently expressed among "nephrons". Also, my strategy to do preliminary dotplots was relied on clustering. Maybe I can get back to marker gene exploration after improving the clustering.


> I think exploring whether the normalization method makes a difference in your analysis could be worth doing; using SCTransform could be a worthwhile comparison. I would not necessarily try to use Azimuth directly though, as we have found that its extra overhead and complexity may cause more challenges than using the more direct anchor transfer functions. I also would not expect major differences, but it would be worth confirming that!

Thanks for pointing out normalization method here. `SCTransform` is way more computationally intensive, particularly in terms of memory usage, when I previously worked on a 2XL Lightsail machine. But I would try it out since I have been moving to a 4XL machine. Also thank you for mentioning the previous experience on Azimuth.

**Comment by sjspielman** [comment]
Thanks for taking the time to detail some of what you have explored here, @JingxuanChen7! I'm going to tag in @maud-p for this conversation too, since you are both working on different Wilms cell type annotation projects and are exploring similar approaches. 

@maud-p, some of the insights here are likely relevant to your analysis as well. Are there any specific similarities or differences you have noticed in your initial runs of CNV methods?

**Reply by maud-p** [reply]
Thank you @sjspielman to point me to this 😄 I see similar results with `copykat` that do not seem to be working for Wilms tumor (very low CNV profiles). I played with different parameters and results do not match (distance = euclidean or spearman, normal cells or not).

Maybe if I may one "warning" about the way of defining the healthy reference in Wilms tumor. Wilms tumor has a tri-phasic histology with three _cancer_ components that are blastema, stroma and epithelial. The three of them are _cancer cells_!! In addition, there might be some reactive stroma that include fibroblats (and CAFs). Tbh I am not sure where there are, but I wouldn't consider the predicted stroma to be all _reactive_ stroma only and not _cancer_ stroma. In Wilms tumor, I would rather use immune and/or endothelial cells as healthy reference. 

It might also be that _all_ fetal nephron predicted cells are indeed cancer cells, and that would explain why we can't see any flatter cnv pattern in here. In the fetal kidney atlas reference, only very differentiated epithelial kidney cells and podocytes should be present in "mature and healthy" kidney. "Cap mesenchyme" are imho restricted to cancer cells. So looking at the predicted cell type from the fetal kidney reference might be the most informative. 

Great that you are looking at `SCEVAN` 😄 FYI, I will try to explore a bit more `infercnv`. Let's keep in touch!

**Reply by sjspielman** [reply]
> It might also be that all fetal nephron predicted cells are indeed cancer cells, and that would explain why we can't see any flatter cnv pattern in here. 

This is actually something that @jaclyn-taroni and I were sort of (casually, and without any evidence necessarily..!) wondering if it might be the case. One way to potentially answer this, or at least investigate patterns which might support or refute this hypothesis, could be to look at marker gene expression across the cells that label transfer annotated as fetal nephron. In particular, we've seen a given sample may have several distinct clusters that appear to be dominated by (if not entirely comprised of) predicted fetal nephrons. What are the differences among those cells? Comparing gene expression of wilms and healthy kidney marker genes, for example, across these groups may be informative. That said, this sort of analysis will also depend heavily on the clusters themselves, so an approach that looks at individual cells may be helpful in situations where clusters are not necessarily optimized.

**Reply by maud-p** [reply]
Just copying the link of the open PR#801 here and [related comment](https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/801#issuecomment-2413419190)

I think that rather than looking at marker genes, I would explore the label transfer from the [human kidney atlas](https://www.kidneycellatlas.org/) (we already performed the transfer from the human fetal kidney atlas, which is a subset and might be enough). Kidney from a _borned_ child should be closer to mature, while Wilms tumor is known to be more resembling to fetal kidney. When we look at the composition of mature versus fetal kidney:
![image](https://github.com/user-attachments/assets/e2a36053-b3e2-4eea-95f6-13c99e58f256)

Cap Mesenchyme (CM, named for some reason mesenchymal cell in the fetal_kidney_reference, predicted.cell.type) is only present in fetal kidney. I checked, [on completion of nephrogenesis, the cap mesenchyme is exhausted](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4675508/). Meaning that cells annotated as cap mesenchyme = "mesenchymal cell" should be cancer cells.

If we now look at more differentiated and functional units of kidney, they are present in different proportion in embryonic and mature kidney. Cells annotated as "kidney cells" or "podocytes" can be either cancer epithelial cells or normal kidney cells. These cells are the more problematic ones for our aim of annotating cancer versus normal. BUT, there are often a relatively small proportion of the total number of cells 😄

**Reply by sjspielman** [reply]
> CM, named for some reason mesenchymal cell in the fetal_kidney_reference, predicted.cell.type

Ah yes, labeling can be tricky like this! You often see different labels for the same cell types (or worse - the same label for different cell types 😬) across references. This is one reason why we often like to use _ontologies_ to label cell types when that information is available, since it can remove some of this ambiguity (in fact, @jashapiro wrote a [blog post about this](https://www.ccdatalab.org/blog/im-terrible-with-names-but-im-using-ontologies-to-try-to-be-better) last year!). Unfortunately I don't believe references we've used for label transfer have associated ontology ids.

**Reply by JingxuanChen7** [reply]
Thanks both @sjspielman and @maud-p for the discussions!

> Maybe if I may one "warning" about the way of defining the healthy reference in Wilms tumor. Wilms tumor has a tri-phasic histology with three cancer components that are blastema, stroma and epithelial. The three of them are cancer cells!! In addition, there might be some reactive stroma that include fibroblats (and CAFs). Tbh I am not sure where there are, but I wouldn't consider the predicted stroma to be all reactive stroma only and not cancer stroma. In Wilms tumor, I would rather use immune and/or endothelial cells as healthy reference.

@maud-p Thank you so much for pointing this out! I re-ran `inferCNV` using immune cells as reference (I only got few endothelium cells annotated), and the reference upper panel looks cleaner than before.

![image](https://github.com/user-attachments/assets/5e342e79-677d-410b-90e8-27ceedac099b)

> Cap Mesenchyme (CM, named for some reason mesenchymal cell in the fetal_kidney_reference, predicted.cell.type) is only present in fetal kidney. I checked, [on completion of nephrogenesis, the cap mesenchyme is exhausted](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4675508/). Meaning that cells annotated as cap mesenchyme = "mesenchymal cell" should be cancer cells.

That's an interesting and reasonable idea. Actually, in my anchor transfer results, majority of "fetal_nephrons" compartment are Cap Mesenchyme, Renal Vesicle (RV), and S-shaped body (SSB) cells, which should present during development but not in the mature kidney. However, I'm not sure if RV and SSB are exhausted or not.

**Reply by maud-p** [reply]
> I re-ran `inferCNV`

Whooo that looks great now! 💪 
You have a clear [1q](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1850292/)  and chr11 loss, which is expected in Wilms tumor [Cresswell et al. ](https://www.thelancet.com/article/S2352-3964(16)30221-3/fulltext). Seems to make sense as chr11 is an early event in Wilms tumor tumorigenesis, and 1q has a more variable evolutionary timing.

**Reply by maud-p** [reply]
> However, I'm not sure if RV and SSB are exhausted or not.

I think for RV and SSB we need to be a bit more careful, as fully differentiated and mature renal subunits are not in the fetal reference, both healthy and cancer epithelial cells might be labeled as RV and SSB. 
Maybe we should not only use the fetal kidney atlas but also the mature, @JingxuanChen7 , @sjspielman ?
However, I am not sure if we really need it, as the CNV profiles tend to show that almost all of the fetal nephron cells have 1q/chr11 alterations. 
❓ I'll ask confirmation to surgeon and/or pathologist colleagues, but I think Wilns tumors is often contained in a capside, so not so mixed with normal kidney cells. I'll keep you updated once I have an answer.

**Reply by JingxuanChen7** [reply]
Thank you so much @maud-p for all these comments. Really helpful!

Regarding the mature kidney atlas, I've tried it as reference dataset a while ago, on sample [SCPCL000849](https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/analyses/cell-type-wilms-tumor-14/plots/01_anchor_transfer_seurat/SCPCL000849_celltype_core.png). My impression was that there were too many low score (< 0.5) predictions, and only proximal tubule related cell types are annotated, which was less informative than fetal atlas.

<details>
  <summary>Here are my old results. UMAP is a bit different since I made some minor modifications during PR.</summary>
  
<img width="504" alt="image" src="https://github.com/user-attachments/assets/e3a196ce-0470-4901-a61c-3a9209147d31">

<img width="529" alt="image" src="https://github.com/user-attachments/assets/cb7accd0-152e-46c4-9dc6-5923f2f7f355">

<img width="538" alt="image" src="https://github.com/user-attachments/assets/5c4916a7-86ee-4943-9183-ffc8981a0cab">


</details>

**Reply by maud-p** [reply]
Thank you for sharing this, I am now even more convinced that using a fetal reference is the way to go!

**Comment by maud-p** [comment]
Adding @georgecresswell to the discussion, as he his much more experienced in Wilms tumor genetics than I 😃 

_Just some background, George is sitting in the office next door and I though might be more constructive asking my questions here than in private._

What do you think about the infercnv profile?
Is the entire chr11 loss reasonable?

**Reply by georgecresswell** [reply]
Hi @maud-p, I've not read the whole history but 11 loss is definitely possible and obviously so is 1q gain, 14q loss is also common. The 1q gain looks rather noisey, I wouldn't expect such a broad range in sizes, but I think that's just the limitation of the inferCNV resolution.

Of note, 11p _copy number neutral_ loss-of-heterozygosity is often very early, the one that leads to IGF2 loss of imprinting and WT1 loss of heterozygosity (if it's mutated), but doesn't change total copy number. You wouldn't see this event without allele-specific copy number analysis. 11 loss is definitely possible but I can't comment on the timing.

**Reply by maud-p** [reply]
Thank you @georgecresswell !
